### DISCUSSION TITLE
Search webparts properties are not being saved

### DISCUSSION THREAD

**Discussion by X1M3N** [microsoft-search/pnp-modern-search#2230] [Q&A]
Hi there,

I recently installed the last version of the package (we previously were in 3.15).
Most of the configuration seems to be working fine.
However, some properties do not seem to get saved.

For example in the columns edition of the search results webpart, I try to specify the "sort fields".
I select the right field, hit save, reopen the "manage columns" popup, and the field is reset.

![image](https://user-images.githubusercontent.com/72074119/178939639-d939dc06-5661-4478-b259-55eddb45c73e.png)

I get the same behavior on the "sort fields" popup and in the refiners (trying to enable multiselection on some refiners).

Anyone encountered this?

**Comment by X1M3N** [comment]
No one ever faced this?
I thought it was related to cache, but cleaning the cache, rebooting, changing browsers, nothing helps...

We are currently stuck with webpart we cannot configure correctly :(

**Reply by FranckyC** [reply]
This is a known issue. It is already fixed in the develop branch. See https://github.com/microsoft-search/pnp-modern-search/issues/2211

**Reply by X1M3N** [reply]
Alright, cool! 
Any idea when it will be available in the main branch?

By the way, the workaround works for the "Manage columns" popup, but not for the "Edit sort settings" popup.
And another point, the link to the document in the search results is not opening a new tab, even if the _blank parameter is specified in the configuration (and is present in the DOM).

**Reply by X1M3N** [reply]
Hey @FranckyC ,

Do you also have a workaround for the sort settings?
![image](https://user-images.githubusercontent.com/72074119/183896516-0dbd96a9-a3da-4520-84ad-7bb4bfe7012b.png)

I am unable to save the "Sort field display name", it gets wiped when I close the popup...
